src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -34,12 +36,20 @@
 
     public static final String VALID_NAME_AMY = "Amy Bee";
     public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NAME_CAT = "Catherine Quek";
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
+    public static final String VALID_PHONE_CAT = "88888888";
     public static final String VALID_EMAIL_AMY = "amy@example.com";
     public static final String VALID_EMAIL_BOB = "bob@example.com";
+    public static final String VALID_EMAIL_CAT = "cat@example.com";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
+    public static final String VALID_ADDRESS_CAT = "Block 369, Cathy Street 2";
+    public static final String VALID_TELEGRAM_CAT = "Catherine_Quek";
+    public static final String VALID_TELEGRAM_DUMMY = "telegram";
+    public static final String VALID_GITHUB_CAT = "Cat-Quek";
+    public static final String VALID_GITHUB_DUMMY = "github";
```

Comment:
I don't think the dummy values should be in between valid values. Perhaps we can put them at the bottom closer to the invalid values

Reply:
After looking at how the dummy values are used, I don't think we need the dummy values since accepting the last value is what we want to show. We can just use INVALID_TELEGRAM_DESC.